tests/Testcontainers.FirebirdSql.Tests/FirebirdSqlContainerTest.cs

```diff
@@ -93,4 +93,13 @@ public FirebirdSqlSysdba()
         {
         }
     }
+
+    [UsedImplicitly]
+    public sealed class FirebirdSqlWaitForDatabase : FirebirdSqlContainerTest
+    {
+        public FirebirdSqlWaitForDatabase()
+            : base(new FirebirdSqlBuilder().WithWaitStrategy(Wait.ForUnixContainer().UntilDatabaseIsAvailable(FirebirdClientFactory.Instance)).Build())
+        {
+        }
+    }
```

Comment:
Is there any reason why you didn’t apply the container fixture to all the tests? I think we can add the xUnit module to the common test project and use the container fixture everywhere (for `IDatabaseContainer`). That should also reduce overhead and improve performance.

Reply:
> Is there any reason why you didn’t apply the container fixture to all the tests?

I just missed some I guess. I will fix it soon for Firebird and Db2. And it will be a huge win for Db2, which is taking over 7 minutes to start on my machine!

> I think we can add the xUnit module to the common test project and use the container fixture everywhere (for IDatabaseContainer).

That's not a good idea. Adding a project reference to `Testcontainers.Xunit.csproj` in the `Testcontainers.Commons` project would prevent testing `Testcontainers.XunitV3.csproj` in other test projects because of xunit incompatibilities between v2 and v3. I purposefully removed the `PackageReference` to `xunit` in #1378.